azure/Kqlmagic/adal_token_cache.py

```diff
@@ -117,14 +148,45 @@ def _query_cache(self, is_mrrt, user_id, client_id):
                 matches.append(v)
         return matches
 
+    def clear_db(self):
+
+        items = self.db.items()
+
+        for token_tuple in items:
+
+            db_key = token_tuple[0] # 'kqlmagicstore/tokens/USER_ID' address
+            data = token_tuple[1] #encrypted token
+            time_created, _unused = Fernet._get_unverified_token_data(data)  #get token timestamp 
+            EXP_TIME = 1000 #in seconds
+            if(time_created <time.time()- EXP_TIME):
```

Comment:
it will be more clear if it will be writted  (time_created + EXP_TIME) < time.time():